src/backend/sqlite/mod.rs

```diff
@@ -24,3 +24,47 @@ impl QuotedBuilder for SqliteQueryBuilder {
         '"'
     }
 }
+
+impl EscapeBuilder for SqliteQueryBuilder {
+    fn escape_string(&self, string: &str) -> String {
+        string
+            .replace('\\', "\\\\")
+            .replace('"', "\\\"")
+            .replace('\'', "''")
+            .replace('\0', "\\0")
+            .replace('\x08', "\\b")
+            .replace('\x09', "\\t")
+            .replace('\x1a', "\\z")
+            .replace('\n', "\\n")
+            .replace('\r', "\\r")
```

Comment:
It seems like that we should not escape those with a `\` but then I don't know what should be the correct way of handling those. May be just ignoring them?

Reply:
I'm a little confused... I thought was one problem with: `\``, I cannot find alternative implementation for `escape` string for sqlite and cannot find about this in documentation. 
@tyt2y3 can you explain to me please)